core/src/main/scala/kafka/common/InconsistentControllerIdException.scala

```diff
@@ -0,0 +1,10 @@
+package kafka.common
+
+/**
+ * Indicates the controllerId stored in logDirs is not consistent across logDirs.
+ */
+class InconsistentControllerIdException(message: String, cause: Throwable) extends RuntimeException(message, cause) {
```

Comment:
There is already an exception for this, kafka.common.InconsistentBrokerMetadataException.  I think we can just use that one rather than creating  a new exception.

Reply:
Yea, I thought of unifying the two. Let me do that